### DISCUSSION TITLE
How to combine drun and run?

### DISCUSSION THREAD

**Discussion by HappyCthulhu** [davatorium/rofi#1634] [Q&A]
I notice that some of my programs are available through run, some - through drun, some - both there and there. Could you explain to me the logic behind these two modes? Where do each of them get the binaries from?

Ideally, I would like to have one mode through which all my applications were available. Is it possible? If yes, how? Combi mode doesnt show all tools, that available from drun and run

**Comment by DaveDavenport** [comment]
Please check the manpage, I think that holds the answers to these questions.

**Reply by DaveDavenport** [reply]
https://github.com/davatorium/rofi/blob/next/doc/rofi.1.markdown#run
https://github.com/davatorium/rofi/blob/next/doc/rofi.1.markdown#drun

https://github.com/davatorium/rofi/blob/next/doc/rofi.1.markdown#usage I think this gives enough to get you to combine drun and run.
